packages/cozy-stack-client/src/ContactsCollection.js

```diff
@@ -31,6 +33,28 @@ class ContactsCollection extends DocumentCollection {
     }
     return col
   }
+
+  /**
+   * Destroys a contact
+   *
+   * If the contact is linked to accounts, it will be trashed instead of being
+   * destroyed.
+   *
+   * @param  {object} contact - Contact to destroy. IT MUST BE THE FULL CONTACT OBJECT
+   * @returns {Promise} - Resolves when contact has been destroyed
+   */
+  async destroy(contact) {
+    const syncData = get(contact, 'cozyMetadata.sync', {})
```

Comment:
Using an external lib just for this seems overkill, why not just `contact?.cozyMetadata?.sync`?